gasp-node/rollup/runtime/src/runtime_config.rs

```diff
@@ -48,16 +48,6 @@ pub mod tokens {
 	pub const RX_TOKEN_ID: TokenId = 0;
 	pub const ETH_TOKEN_ID: TokenId = 1;
 
-	#[cfg(any(feature = "unlocked", feature = "runtime-benchmarks"))]
```

Comment:
did u disable the -fast config as well?

Reply:
should we create a migration to remove the TransferAccounts and arbitrage account?

Reply:
>>did u disable the -fast config as well?
not sure if i got that, basically there are no nontransferable tokens anymore, same as there are no exceptions for arbitrage bot accoutn